docs/REST API/rest-api.md

```diff
@@ -4859,22 +4859,19 @@ Where `query_id` references an existing `query`.
 }
 ```
 
-### Run Automation for all failing hosts of a policy.
+### Run automation for all failing hosts of a policy
 
-Normally automations (Webhook/Integrations) runs on all hosts when a policy-check
-fails but didn't fail before. This feature to mark policies to call automation for
-all hosts that already fail the policy, too and possibly again.
+Triggers [automations](https://fleetdm.com/docs/using-fleet/automations#policy-automations) for *all* hosts with failing policies, regardless of whether a policy was previously failing on a host.
```

Comment:
I think this line says something different, if I understand correctly. Should it instead be...

```suggestion
Triggers [automations](https://fleetdm.com/docs/using-fleet/automations#policy-automations) for *all* hosts that have the specified policy, regardless of whether the policy was previously failing on a host.
```